milvus/datadog_checks/milvus/metrics.py

```diff
@@ -0,0 +1,226 @@
+    'filter_connectivity_ratio': 'filter.connectivity_ratio',
+    'filter_mv_activated_fields_cnt': 'filter.mv.activated_fields_cnt',
+    'filter_mv_change_base_cnt': 'filter.mv.change_base_cnt',
+    'filter_mv_only_cnt': 'filter.mv.only_cnt',
+    'filter_mv_supplement_ep_bool_cnt': 'filter.mv.supplement_ep_bool_cnt',
+    'go_gc_duration_seconds': 'go.gc_duration_seconds',
+    'go_goroutines': 'go.goroutines',
+    'go_info': 'go.info',
+    'go_memstats_alloc_bytes': 'go.memstats.alloc_bytes',
+    # 'go_memstats_alloc_bytes_total': 'go.memstats.alloc_bytes_total',
+    'go_memstats_buck_hash_sys_bytes': 'go.memstats.buck_hash_sys_bytes',
+    'go_memstats_frees': 'go.memstats.frees',
+    'go_memstats_gc_sys_bytes': 'go.memstats.gc_sys_bytes',
+    'go_memstats_heap_alloc_bytes': 'go.memstats.heap.alloc_bytes',
+    'go_memstats_heap_idle_bytes': 'go.memstats.heap.idle_bytes',
+    'go_memstats_heap_inuse_bytes': 'go.memstats.heap.inuse_bytes',
+    'go_memstats_heap_objects': 'go.memstats.heap.objects',
+    'go_memstats_heap_released_bytes': 'go.memstats.heap.released_bytes',
+    'go_memstats_heap_sys_bytes': 'go.memstats.heap.sys_bytes',
+    'go_memstats_last_gc_time_seconds': 'go.memstats.last_gc_time_seconds',
+    'go_memstats_lookups': 'go.memstats.lookups',
+    'go_memstats_mallocs': 'go.memstats.mallocs',
+    'go_memstats_mcache_inuse_bytes': 'go.memstats.mcache.inuse_bytes',
+    'go_memstats_mcache_sys_bytes': 'go.memstats.mcache.sys_bytes',
+    'go_memstats_mspan_inuse_bytes': 'go.memstats.mspan.inuse_bytes',
+    'go_memstats_mspan_sys_bytes': 'go.memstats.mspan.sys_bytes',
+    'go_memstats_next_gc_bytes': 'go.memstats.next_gc_bytes',
+    'go_memstats_other_sys_bytes': 'go.memstats.other_sys_bytes',
+    'go_memstats_stack_inuse_bytes': 'go.memstats.stack.inuse_bytes',
+    'go_memstats_stack_sys_bytes': 'go.memstats.stack.sys_bytes',
+    'go_memstats_sys_bytes': 'go.memstats.sys_bytes',
+    'go_threads': 'go.threads',
+    'graph_search_cnt': 'graph_search_cnt',
+    'hnsw_bitset_ratio': 'hnsw.bitset_ratio',
+    'hnsw_search_hops': 'hnsw.search_hops',
+    'internal_core_search_latency': 'internal.core_search_latency',
+    'internal_mmap_allocated_space_bytes': 'internal.mmap.allocated_space_bytes',
+    'internal_mmap_in_used_space_bytes': 'internal.mmap.in_used_space_bytes',
+    'internal_storage_kv_size': 'internal.storage.kv_size',
+    'internal_storage_load_duration': 'internal.storage.load_duration',
+    'internal_storage_op_count': 'internal.storage.op',
+    'internal_storage_request_latency': 'internal.storage.request_latency',
+    'io_cnt': 'io_cnt',
+    'ivf_search_cnt': 'ivf_search_cnt',
+    'load_latency': 'load_latency',
+    'process_cpu_seconds': 'process.cpu_seconds',
+    'process_max_fds': 'process.max_fds',
+    'process_open_fds': 'process.open_fds',
+    'process_resident_memory_bytes': 'process.resident_memory_bytes',
+    'process_start_time_seconds': 'process.start_time_seconds',
```

Comment:
```suggestion
    'process_start_time_seconds': {'name': 'process.start_time_seconds', 'type': 'time_elapsed'},
```

Probably a better use of this metric.